### PR TITLE
adjust ocean location, fix path for ocean checkmark

### PR DESCRIPTION
**Pull request by vsrisuknimit**
- Fix path to ocean checkmark.
- Adjust the ocean location, number and check mark to be more center.

### CODE REVIEW

**Comment by vsrisuknimit** [comment]
@alrusdi

File: src/components/Board.ts [hunk 1]
```diff
@@ -111,8 +111,8 @@ export const Board = Vue.component("board", {
         oceansValue: function() {
             const oceans_count = this.oceans_count || 0;
             const leftover = constants.MAX_OCEAN_TILES - oceans_count;
-            if (leftover === 0) {
-                return `<img width="26" src="/assets/circle-checkmark.png" :alt="$t('Completed!')">`
+            if (leftover === 8) {
```
Comment (alrusdi) [inline src/components/Board.ts:114]: It's probably wrong!

**Review by alrusdi — CHANGES_REQUESTED** [review]

File: src/components/Board.ts (hunk 1 above)
Comment (vsrisuknimit) [reply src/components/Board.ts]: sorry. I added that to debug and forgot to remove it. Very sloppy. My bad.

**Comment by alrusdi** [comment]
it will rewrite new image of mars background. I am rejecting it.

**Pull request closed without merging**